source/main.c

```diff
@@ -634,6 +635,21 @@ static int force_disc() {
 	return type;
 }
 
+/*
+ Detect if a dual-layer disc was inserted by checking if reading from sectors
+ on the second layer is succesful or not.
+*/
+int detect_duallayer_disc() {
+	char *readBuf = (char*)memalign(32,64*1);
```

Comment:
what's with the 64*1? just make it 64 :)
```suggestion
	char *readBuf = (char*)memalign(32,64);
```

Reply:
fixed.